test.cfg.example

```diff
@@ -0,0 +1,39 @@
+[HandleService2]
+
+# the path to the MongoDB executable to use.
+test.mongo.exe=mongod
+
+# whether to use the wired tiger storage engine for MongoDB. 'true' means yes, anything else means
+# no.
+test.mongo.wired_tiger=false
+
+# Where to store temporary files generated during the test.
+test.temp.dir=temp_test_dir
+
+# Whether test files should be discarded after testing is complete.
+# Anything other than 'false' is considered true.
+test.delete.temp.dir=true
+
+# The root url of the auth server.
+auth-service-url=
+
+# The KBase authorization service mkII url.
+auth-url=
+
+# The root url of the shock server.
+shock-url=
+
+# Admin token
+admin-token=
+
+# Admin roles
+admin-roles=HANDLE_ADMIN, KBASE_ADMIN
+
+# A prefix appended to handle id
+namespace=KBH
+
+# Mongo database name
+mongo-database=handle_db
+
+# Scratch url
+scratch=/kb/module/work/tmp
```

Comment:
This won't be available outside of a docker container, what is it used for in testing?

Reply:
ah, is assigned in AbstractHandle_server_test.py, but nerver used. Maybe I just delete it?

Reply:
👍

Reply:
:+1: